docs/how_to/create_a_quantum_circuit.rst

```diff
@@ -0,0 +1,167 @@
+
+########################
+Create a quantum circuit
+########################
+
+This guide shows how to initialize a quantum circuit.
+
+There are two ways to create a :class:`~.QuantumCircuit` object:
+
+* Specifying the number of qubits and bits.
+* Creating :class:`~.QuantumRegister`\ s and :class:`~.ClassicalRegister`\ s.
+
+Create from number of qubits and bits
+=====================================
+
+In order to create a :class:`~.QuantumCircuit` by only specifying the number of bits and qubits, you need to follow these steps.
+
+.. testcode::
+
+    from qiskit import QuantumCircuit
+
+    # Initialize number of qubits and classical bits
+    n_qubits = 3
+    n_bits = 2
+
+    # Create the circuit
+    qc = QuantumCircuit(n_qubits, n_bits)
+    print(qc.draw())
+
+.. testoutput::
+    :options: +NORMALIZE_WHITESPACE
+
+    q_0: 
+        
+    q_1: 
+        
+    q_2: 
+        
+    c: 2/
+     
+
+If you don't want to include any classical bits, you don't have to write `QuantumCircuit(n_qubits,0)` but you can omit the number of classical bits.
+
+.. testcode::
+
+    from qiskit import QuantumCircuit
+
+    qc = QuantumCircuit(n_qubits)
+    print(qc.draw())
+
+.. testoutput::
+    :options: +NORMALIZE_WHITESPACE
+
+    q_0: 
+        
+    q_1: 
+        
+    q_2: 
+
+Create from quantum and classical registers
+===========================================
```

Comment:
A how to guide should provide tips for users to decide how to choose different options of the same task. Can you comment on when people should use this method and when they should use the simpler method with specifying only number of qubits and classical bits? For example, the simpler method can only create circuits with one quantum register and one classical register. And it also can't specify names to the registers.